src/components/pages/Dashboard/Explore.tsx

```diff
@@ -1,79 +1,113 @@
-            </div>
-          </div>) :
-          (
-            <div className="fill flex">
-              <MobileComponents />
+    return (
+        <>
+            {
+                //I know we prefer classes to inline styles, I will upgrade the same once utility classes from my punlication
+                //PR are available
+            }
+            <div className="fill flex dashboard">
+                {!isMobileDeviceOrTablet ?
+                    (<>
+                        <div className={showUnityBanner ? "px-4 flex center-item space-between w-100 visible" : "invisible"} style={{backgroundColor: "#E8EBEF", height: showUnityBanner ? "50px" : "0"}}>
+                            <div style={{width: "80%"}}>
+                                {Translate("UnityBanner", ["Currently"])}
+                              <a href={"https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/early-investigations"}
+                                 target="_blank" rel="noopener noreferrer">
+                                {Translate("UnityBanner", ["ProtocolLink"])}
+                              </a> {Translate("UnityBanner",["ViewDatabase"])} <span className={"a"} onClick={() => {setPulsateUnityCheckbox(true)}}> {Translate("UnityBanner", ["WHOUnityFilter"])} </span>
+                                {Translate("UnityBanner", ["Filter"])}
+                            </div>
+                            <Icon link onClick={() => {setShowUnityBanner(false)}} name='close'/>
+                        </div>
+                        <div className="flex w-100">
+                            <div className="col-2 p-0 flex">
+                                <LeftSidebar page={PageStateEnum.explore}/>
+                            </div>
+                            <div className="col-8 p-0 flex" id={PAGE_HASHES.Explore.Map}>
+                                <Loader indeterminate active={state.explore.isLoading}/>
+                                <div className="info flex legend center-item">
+                                    <Legend/>
+                                </div>
+                                <MapboxMap
+                                    countriesConfig={{
+                                        estimateGradePrevalences: state.explore.estimateGradePrevalences
+                                    }}
+                                    studyPinsConfig={{
+                                        records: state.explore.records
+                                    }}
+                                />
+                            </div>
+                            <div className="col-2 p-0 flex sidebar-container">
+                                <Filters page={PageStateEnum.explore}  pulsateUnityFilter={pulsateUnityCheckbox} setPulsateUnityFilter={setPulsateUnityCheckbox}/>
+                            </div>
+                        </div>
+                    </>) :
+                    (
+
```

Comment:
@atatmaja @annoyingpr @brett-stephen note, I auto aligned the whole while working due to habit, and so I am just marking where the actual changes I made were --- here to the bottom v